lib/kennel/models/monitor.rb

```diff
@@ -238,6 +239,8 @@ def validate_json(data)
           validate_message_variables(data)
         end
 
+        validate_using_links(data)
```

Comment:
Prefer a boolean setting:

```suggestion
        validate_using_links(data) if require_using_links
```

Having a "setting" where the two values are "method with logic" or "override with a no-op block" is ... a weird way of programming.

Reply:
Particularly when the method/block in question effectively returns void

Reply:
yeah it's not very smooth ... some more general `skip_validations: [:foo]` might be nicer ... but that's more refactoring and can be it's own PR

Reply:
another option would be
```
eval "->(*) { v }", nil, file, line.to_i
```
to use `validate_using_links: false`
but that ends up making any setting callable which could end up ugly too